app/src/main/kotlin/at/bitfire/davdroid/webdav/cache/DiskCache.kt

```diff
@@ -95,7 +95,7 @@ class DiskCache(
         files.sortBy { file -> file.lastModified() }    // sort by modification time (ascending)
 
         while (files.sumOf { file -> file.length() } > maxSize) {
-            val file = files.removeFirst()      // take first (= oldest) file
```

Comment:
Yes, let's use `removeAt(0)`.

Just for curiosity I tried to find out how to force the Kotlin stdlib (no minimum SDK) instead of Java List `removeFirst` (which has a minimum SDK) and the only thing I got to work is:

```
import kotlin.collections.removeFirst as removeFirstKotlin

val file = files.removeFirstKotlin()
```

But it does `removeAt(0)` anyway, so let's use that :)